.travis.yml

```diff
@@ -1,56 +1,19 @@
+sudo: false
 language: php
 
-## Cache composer bits
-cache:
-  directories:
-    - $HOME/.composer/cache/files
-
 ## Build matrix for lowest and highest possible targets
 matrix:
   include:
-    - php: 7.0
-      env:
-        - qaExtended=true
-    - php: 7.1
-    - php: nightly
-      env:
-        - dropPlatform=false
-    - php: 7.0
-      env:
-        - dependencies=lowest
-    - php: 7.1
-      env:
-        - dependencies=lowest
     - php: nightly
-      env:
-        - dependencies=lowest
-        - dropPlatform=false
-    - php: 7.0
-      env:
-        - dependencies=highest
-    - php: 7.1
-      env:
-        - dependencies=highest
-    - php: nightly
-      env:
-        - dependencies=highest
-        - dropPlatform=false
 
 ## Install or update dependencies
 install:
   - composer validate
-  - if [ -z "$dropPlatform" ]; then composer config --unset platform.php; fi;
```

Comment:
I wonder whether it is caused to some configuration issues. What is this doing ?

Reply:
Interesting, so removing that solved the issue 0_o? So what that does is unset the following from `composer.json` 

```json
{
  "config": {
    "platform": {
      "php": "7.0"
    }
}
```

That is there to ensure no packages (or versions of them) cam be installed using composer require. For example my local `PHP` version is `7.2` and packages like `doctrine/annotation` have `^7.1` and maybe even `^7.2` versions available. When running `composer require doctrine/annotation` a version is selected that can be used on `7.0` even though my local verion is `7.2`.